javascript/kadai/1-4/1-4.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * 課題①-4
+ */
+
+const tasks = [
+  {
+    content: "机を片付ける",
+    genre: "掃除",
+  },
+  {
+    content: "牛乳を買う",
+    genre: "買い物",
+  },
+  {
+    content: "運動",
+    genre: "散歩する",
+  },
+];
+
+showTaskList();
+const content = window.prompt("タスクを入力してください");
+const genre = window.prompt("ジャンルを入力してください");
+const task = {
+  content: content,
+  genre: genre,
+};
```

Comment:
オブジェクトは、keyとvalueが同じ命名の場合、以下のように省略して記述できます。
```suggestion
const task = {
  content,
  genre,
};
```
この省略記法はすっきり書けることからとてもよく使用されるので、今回のように省略できるよう命名していくのも大切です！

Reply:
>オブジェクトは、keyとvalueが同じ命名の場合、以下のように省略して記述できます。

これ結構びっくりでした！
人が書いたコードを見る際気を付けておきます。